modules/storages/spec/features/storages/admin/project_storages_spec.rb

```diff
@@ -31,35 +31,24 @@
 require "spec_helper"
 require_module_spec_helper
 
+# We decrease the notification polling interval because some portions of the JS code rely on something triggering
+# the Angular change detection. This is usually done by the notification polling, but we don't want to wait
 RSpec.describe "Admin lists project mappings for a storage",
                :js,
+               :webmock,
                :with_cuprite,
-               with_flag: { enable_storage_for_multiple_projects: true } do
+               with_flag: { enable_storage_for_multiple_projects: true },
+               with_settings: { notifications_polling_interval: 1_000 } do
   shared_let(:admin) { create(:admin, preferences: { time_zone: "Etc/UTC" }) }
   shared_let(:non_admin) { create(:user) }
 
   shared_let(:project) { create(:project, name: "My active Project") }
-  shared_let(:archived_project) do
-    create(:project,
-           active: false,
-           name: "My archived Project")
-  end
-  shared_let(:storage) do
-    create(:nextcloud_storage,
-           :as_automatically_managed,
-           name: "My Nextcloud Storage")
-  end
-  shared_let(:project_storage) do
-    create(:project_storage,
-           project:,
-           storage:,
-           project_folder_mode: "automatic")
-  end
+  shared_let(:archived_project) { create(:project, active: false, name: "My archived Project") }
+  shared_let(:storage) { create(:nextcloud_storage_with_complete_configuration, name: "My Nextcloud Storage") }
+  shared_let(:project_storage) { create(:project_storage, project:, storage:, project_folder_mode: "automatic") }
```

Comment:
I actually did make a change by using `:nextcloud_storage_with_complete_configuration` 😄  But I do prefer the collapsed structure to reduce the perceived test setup fatigue.

Reply:
> I actually did make a change by using :nextcloud_storage_with_complete_configuration

Ah, I totally missed that on the diff because of the other changes. Thanks for pointing it out and explaining it! :+1: 

As I said, I don't care too much and we can keep it that way. I personally find it easier to see which attributes are set and what the differences between the objects are if the attributes are in a new line. So it adds readability to me personally. And the global test setup should always be short anyways, so that it's easy to add new test cases which might need a different setup. So the newlines and the scrolling fatigue are like added motivation for that :smile: 
I had a discussion about that with Kabiru in personal before, so I didn't explain it in my comment again :wink: